documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Metadata_Broker.md

```diff
@@ -1,12 +1,17 @@
 ### Metadata Broker ###
 
-The IDS Metadata Broker consists of an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), an endpoint for the registration, publication, maintenance, and query of Self-Descriptions. Therefore, for any interaction with the IDS Metadata Broker, the processes defined on the Process Layer, the descriptions defined on the Information Layer, and descriptions defined on the System Layer can be applied. The Information Layer describes the message types for registration and query. An IDS Metadata Broker may provide additional services that in term must be described by using terms from the IDS Information Model in the respective Metadata Broker's Self-Description document.
+The IDS Metadata Broker consists of an IDS Connector (see Section [3.5.2](./3_5_2_IDS_Connector.md#ids-connector)), an endpoint for the registration, publication, maintenance, and query of Self-Descriptions. 
+
+A Self-Description encapsulates information about the component hosted within an IDS Connector. This Self-Description contains information about the offered interfaces, the owner of the component and the metadata of the data offered by the component. A Self-Description is provided by the owner of a component. The Self-Description in total can be seen as metadata.<br>
```

Comment:
A Self-Description encapsulates information about the IDS Connector itself and its capabilities and characteristics. ... A Self-Description is provided by the operator of the Connector. ...